projekte/2015-06-12-be-oparl.html

```diff
@@ -1,21 +1,22 @@
 ---
 layout: project
-lab: OK Lab Berlin
+lab: OK Labs Berlin, München, Ruhrgebiet
```

Comment:
In dem Projekte-Code ist eh der Wurm drinnen.  Wenn englische Beschreibungen vorliegen, tauchen die Projekte doppelt auf.  Vgl. `München Transparent` in http://codefor.de/projekte/alle/.  Gleiches bei http://codefor.de/en/projekte/alle/.